monitoring/grafana/dashboards/jsonnet/grafana_dashboards.jsonnet

```
@@ -494,7 +494,7 @@ local addStyle(alias, colorMode, colors, dateFormat, decimals, mappingType, patt
         'Latencies are shown stacked, without a yaxis to provide a visual indication of GET latency imbalance across RGW hosts',
         's',
         'short',
-        'label_replace(rate(ceph_rgw_get_initial_lat_sum[30s]),"rgw_host","$1","ceph_daemon","rgw.(.*)") / \nlabel_replace(rate(ceph_rgw_get_initial_lat_count[30s]),"rgw_host","$1","ceph_daemon","rgw.(.*)")',
+        'label_replace(\n    rate(ceph_rgw_get_initial_lat_sum[30s]) /\n    rate(ceph_rgw_get_initial_lat_count[30s]) *\n    on (instance_id) group_left (ceph_daemon) ceph_rgw_metadata,\n"rgw_host", "$1", "ceph_daemon", "rgw.(.*)")',
```

Let's better use jsonnet verbatims (`|||`) instead of explicit line breaks:
```suggestion
        |||
        label_replace(
            rate(ceph_rgw_get_initial_lat_sum[30s]) /
            rate(ceph_rgw_get_initial_lat_count[30s]) *
            on (instance_id) group_left (ceph_daemon) ceph_rgw_metadata,
            "rgw_host", "$1", "ceph_daemon", "rgw.(.*)")
        |||,
```